parts/k8s/kubernetescustomscript.sh

```diff
@@ -67,6 +67,7 @@ installNetworkPlugin
 installContainerd
 extractHyperkube
 ensureRPC
+createKubeManifestDir
```

Comment:
Note: if we *do* decide to keep this in CSE, let's move this fun invocation into the `if $FULL_INSTALL_REQUIRED; then` block. We should create this directory during VHD image creation process.

Reply:
`if $FULL_INSTALL_REQUIRED;` is for deployment not using the AKS image so I don't think we should move it, we want this to run on every deployment